slack_bolt/context/base_context.py

```diff
@@ -103,13 +105,23 @@ def matches(self) -> Optional[Tuple]:
         """Returns all the matched parts in message listener's regexp"""
         return self.get("matches")
 
+    @property
+    def function_execution_id(self) -> Optional[str]:
+        """The `function_execution_id` associated with this request."""
```

Comment:
Should we document that this is only available for `function_executed` events and interactivity events scoped to a function?

Reply:
Good point I will add something like "only available for function related events"?